api/libproject/devstart.rb

```diff
@@ -468,7 +468,8 @@ def get_test_service_account_creds(*args)
   :description => \
     "Removes docker containers and volumes, allowing the next `dev-up` to" \
     " start from scratch (e.g., the database will be re-created). Includes ALL" \
-    " docker images, not just for the API.",
+    " docker images, not just for the API. Warning: You will probably have to" \
```

Comment:
Why would you need to reboot? 

I know you ran into that problem once, but I have run this target a bunch of times and never had that issue...

Reply:
Just saw your comment. Weird.

Reply:
Maybe it's `Mark will probably` and not generic `You`. I'm getting the same error as before.